code/modules/client/preferences.dm

```diff
@@ -367,6 +368,15 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 					dat += "[features["flavor_text"]]"
 			else
 				dat += "[TextPreview(features["flavor_text"])]...<BR>"
+			dat += "<BR>"			
```

Comment:
use the same format as above please, h2 instead of br, it'll look weird otherwise